vortexasdk/api/breakdown_item.py

```diff
@@ -0,0 +1,20 @@
+from dataclasses import dataclass
+from typing import Dict, List
+
+from vortexasdk.api import ISODate
+from vortexasdk.api.serdes import FromDictMixin
+
+
+@dataclass(frozen=True)
+class BreakdownItem(FromDictMixin):
+    """
+    Generic container class holding a _key_, _value_ pair, and a _count_ and breakdown of records contributing to the given value.
+
+    For example, this class could hold the average speed of vessels (value) on 2019-01-01 (key), the number of vessels contributing to the
+    this average (count) and additional information about the aggregation (breakdown).
+    """
+
+    key: ISODate
```

Comment:
What about making key `string ` and the breakdown field optional?
This way we would remove the need to have a separate geo_breakdown type

Reply:
P.s. this if neither breakdown or geo_breakdown support paging